### PR TITLE
Standardize http metrics, to service_request_duration

### PR DESCRIPTION
**Pull request by paulbellamy**
Should be the same as we use for other services.

### CODE REVIEW

**Comment by paulbellamy** [comment]
To match queries in: https://github.com/weaveworks/monitoring/pull/109

File: cmd/fluxsvc/main.go [hunk 1]
```diff
@@ -99,21 +99,20 @@ func main() {
...
+		httpDuration     *stdprometheus.HistogramVec
 		instanceMetrics  instance.Metrics
 		jobWorkerMetrics jobs.WorkerMetrics
 		registryMetrics  registry.Metrics
 		releaseMetrics   release.Metrics
 		serverMetrics    server.Metrics
 	)
 	{
-		httpDuration = prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
+		httpDuration = stdprometheus.NewHistogramVec(stdprometheus.HistogramOpts{
```
Comment (squaremo) [inline cmd/fluxsvc/main.go]: One thing this loses is the request _counts_ (and therefore rates). Does the standard instrumentation not care about those?

Comment (paulbellamy) [inline cmd/fluxsvc/main.go]: ? This doesn't lose that?

Comment (paulbellamy) [inline cmd/fluxsvc/main.go]: for a histogram prom creates the three: `flux_request_duration_seconds_(count|bucket|sum)`

Comment (squaremo) [inline cmd/fluxsvc/main.go]: I beg your puddin', I thought that was something gokit's metrics did. As you were ...

**Pull request merged**